### PR TITLE
Add MA styling with bower

### LINKED ISSUE #19: Use bower for makers_styles in careers front-end
Should we use the makers_styles in bower

### PR DESCRIPTION
**Pull request by spike01**
Closes #19

### CODE REVIEW

**Comment by spike01** [comment]
Still doesn't look great with the flash notices, but... hey.

File: README.md [hunk 1]
```diff
@@ -3,3 +3,16 @@
 [ ![Codeship Status for makersacademy/careers](https://codeship.com/projects/d0e1d050-3d17-0133-c9b5-22d459b325ce/status?branch=master)](https://codeship.com/projects/102380)
 
 The location for students to finish jobs, written in Rails
+
+## Installation
+
+* Clone the repo
+* `bin/rake db:setup`
+* `bin/rake db:seed`
```
Comment (pitchinvasion) [inline README.md]: Add a note about running bower install

**Comment by pitchinvasion** [comment]
We still need it to deploy on Heroku

https://gist.github.com/afeld/5704079/

Also CI needs to work with bower no?

**Comment by spike01** [comment]
CI is done (thus the passing build) and deploy happens when there's a squirrel.

**Comment by pitchinvasion** [comment]
cool :shipit:

**Pull request merged**